andela/codepirates-ah-frontend
# #167313370 Share articles across social platforms

#### What does this PR do?

Add the capability to share articles across facebook, twitter and by email

#### Description of Task to be completed?

- Embed a functional component with social sharing links in the article view page
- Listen to click events on each of the sharing icons to send a sharing request to the back-end

#### How should this be manually tested?

- Clone this repo and run the server; check appropriate script in package.json.scripts
- Log in to an existing account or signup and then login
- Select an article on the landing page and click on any of the sharing icons at the foot of the article
- Follow the interactive prompts to complete the sharing in the browser window that opens
- When complete, open the forum you shared in and click on the shared link
- You should see the very page you shared


#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?

[WIP #167313370](https://www.pivotaltracker.com/story/show/167313370)

#### Screenshots (if appropriate)

#### Questions:
